testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OIDCProtocolMappersTest.java

```diff
@@ -594,6 +594,51 @@ public void testRolesAndAllowedOriginsRemovedFromAccessToken() throws Exception
         }
     }
 
+    /**
+     * Chris Nurse 20-Feb-2022
+     * 
+     * PR-4205 serialised single values as an array. This is not desired when the Multivalued options is set to false.
+     * 
+     * This test ensures singular values are not wrapped as an array.
+     * 
+     * The subsequent test for PR-4205 will continue to test functionality where multivalued = true
```

Comment:
Instead of naming a PR and a relative position, use the name of the test, as positions in a file may change over time.